code/game/gamemodes/changeling/powers/biodegrade.dm

```diff
@@ -22,12 +22,14 @@
 			return 0
 		user.visible_message("<span class='warning'>[user] vomits a glob of acid on \his [O]!</span>", \
 							 "<span class='warning'>We vomit acidic ooze onto our restraints!</span>")
-		spawn(30)
-			if(O && user.handcuffed == O)
-				user.unEquip(O)
-				O.visible_message("<span class='warning'>[O] dissolves into a puddle of sizzling goop.</span>")
-				O.loc = get_turf(user)
-				qdel(O)
+
+		sleep(30)
+
+		if(O && user.handcuffed == O)
+			user.unEquip(O)
+			O.visible_message("<span class='warning'>[O] dissolves into a puddle of sizzling goop.</span>")
+			O.loc = get_turf(user)
+			qdel(O)
```

Comment:
This would cause the proc to block and not run the stuff below this while it slept. 

These spawns were most likely spawns for a reason.